src/wheel/vendored/packaging/_manylinux.py

```diff
@@ -236,7 +236,7 @@ def _is_compatible(name: str, arch: str, version: _GLibCVersion) -> bool:
         return False
     # Check for presence of _manylinux module.
     try:
-        import _manylinux  # noqa
+        import _manylinux
```

Comment:
Ahh, this is vendored. Yuck, needs to be fixed there then. I can add vendored to the ignore list for flake8.

Reply:
We will switch to a real logger with the other PR.

Reply:
I've added the vendored directory to the global ignore and revered the change here, so pre-commit won't trigger on things inside it. And, yes, for the other thing, a real logger would be better :).